policykit/policyengine/models.py

```diff
@@ -216,8 +216,14 @@ class GenericRole(Group):
     is_base_role = models.BooleanField(default=False)
 
     user_group = models.TextField(blank=True, null=True, default='')
-
-    def make_community_role(self, community, creator_token=None):
+    
+    constitution_perms = models.TextField(blank=True, null=True, default='')
```

Comment:
don't think we should be doing this (storing via json, and having specific fields for different types of perms)